js/src/ui/IdentityIcon/identityIcon.js

```diff
@@ -65,16 +66,29 @@ export default class IdentityIcon extends Component {
       return;
     }
 
+    let scale = 7;
+    if (button) {
+      scale = 3;
+    } else if (inline) {
+      scale = 4;
+    }
+
     this.setState({
-      iconsrc: api.util.createIdentityImg(_address, inline ? 4 : 7)
+      iconsrc: api.util.createIdentityImg(_address, scale)
     });
   }
 
   render () {
-    const { className, center, inline, padded } = this.props;
+    const { button, className, center, inline, padded } = this.props;
     const { iconsrc } = this.state;
-    const size = inline ? '32px' : '56px';
-    const classes = `${styles.icon} ${center ? styles.center : styles.left} ${padded ? styles.padded : ''} ${inline ? styles.inline : ''} ${className}`;
+    const classes = `${styles.icon} ${center ? styles.center : styles.left} ${padded ? styles.padded : ''} ${inline ? styles.inline : ''} ${button ? styles.button : ''} ${className}`;
```

Comment:
Like that, should have done it from the start - but kept adding more and more cruft. Will fix.